### PR TITLE
feat(pgbouncer): set postgresql database as pgbouncer database if it's not set

### PR DESCRIPTION
**Pull request by ihor-hrytskiv**
### Description of the change

Set `POSTGRESQL_DATABASE` as `PGBOUNCER_DATABASE` if it's not set

### Benefits

Less configuration errors because if we set `PGBOUNCER_DATABASE` and do not set `POSTGRESQL_DATABASE` dbname in _pgbouncer.ini_ will be postgres. It is not good, pgbouncer maps `PGBOUNCER_DATABASE` to postgres and the program writes its data to the wrong database.

### CODE REVIEW

File: bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh [hunk 1]
```diff
@@ -166,7 +166,7 @@ export PGBOUNCER_CLIENT_IDLE_TIMEOUT="${PGBOUNCER_CLIENT_IDLE_TIMEOUT:-}"
 # PostgreSQL backend settings
 export POSTGRESQL_USERNAME="${POSTGRESQL_USERNAME:-postgres}"
 export POSTGRESQL_PASSWORD="${POSTGRESQL_PASSWORD:-}"
-export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-postgres}"
+export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-${PGBOUNCER_DATABASE:-postgres}}"
```
Comment (jotamartos) [inline bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh:169]: As PGBOUNCER_DATABASE is already set, you can use this

```suggestion
export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-${PGBOUNCER_DATABASE}}"
```

What do you think?

**Review by jotamartos — CHANGES_REQUESTED** [review]
Thanks for taking the time to create this PR. Please take a look at my comment, I think we can simplify the logic

File: bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh (hunk 1 above)
Comment (ihor-hrytskiv) [reply bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh:169]: yes, I think it's better

File: bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh [hunk 2]
```diff
@@ -166,7 +166,7 @@ export PGBOUNCER_CLIENT_IDLE_TIMEOUT="${PGBOUNCER_CLIENT_IDLE_TIMEOUT:-}"
 # PostgreSQL backend settings
 export POSTGRESQL_USERNAME="${POSTGRESQL_USERNAME:-postgres}"
 export POSTGRESQL_PASSWORD="${POSTGRESQL_PASSWORD:-}"
-export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-postgres}"
+export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-${PGBOUNCER_DATABASE}}"
```
Comment (jotamartos) [inline bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh:169]: Sorry, there was a typo in my previous suggestion

```suggestion
export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-${PGBOUNCER_DATABASE}"
```

**Review by jotamartos — CHANGES_REQUESTED** [review]
Please fix the typo. The tests should succeed after that.

File: bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh (hunk 2 above)
Comment (jotamartos) [reply bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh:169]: Sorry, this is correct. I thought there was an extra } character.

**Review by jotamartos — APPROVED** [review]

**Comment by jotamartos** [comment]
New version of the container is now released with your changes

https://github.com/bitnami/containers/pull/37018

**Pull request merged**